### PR TITLE
[WIP] Update to release 5.4.1

### PR DESCRIPTION
**Pull request by deki**
Update to latest Identity server release

### CODE REVIEW

**Comment by chirangaalwis** [comment]
@deki Thanks for your contribution! Can you resolve the conflicts and resend the PR for consideration?

**Comment by deki** [comment]
Well looks like the docker-compose stuff got restructured, I have to test the changes first.

**Comment by DilanUA** [comment]
@deki, Yes, we have been doing some considerable restructuring and we have to test for Identity 5.4.1 release now (https://docs.wso2.com/display/IS541/WSO2+Identity+Server+Documentation).

Thanks.

**Comment by DilanUA** [comment]
@deki Can you resend the PR just for dockerfiles section considering 5.4.1 version of Identity Server, since we are already working on the compose side ?

Thanks.

File: docker-compose/is-analytics/identity-server-analytics/repository/conf/carbon.xml [hunk 1]
```diff
@@ -36,7 +36,7 @@
     <!--
        Product Version
     -->
-    <Version>5.3.0-SNAPSHOT</Version>
+    <Version>5.4.1</Version>
```
Comment (DilanUA) [inline docker-compose/is-analytics/identity-server-analytics/repository/conf/carbon.xml]: @deki Ideally we should be using the configuration file from 5.4.1 default pack for this and update that with any deployment specific configurations.

File: docker-compose/is-analytics/identity-server/repository/conf/carbon.xml [hunk 2]
```diff
@@ -36,7 +36,7 @@
     <!--
        Product Version
     -->
-    <Version>5.3.0</Version>
+    <Version>5.4.1</Version>
```
Comment (DilanUA) [inline docker-compose/is-analytics/identity-server/repository/conf/carbon.xml]: @deki Ideally we should be using the configuration file from 5.4.1 default pack for this and update that with any deployment specific configurations.

**Comment by DilanUA** [comment]
@deki Thank you very much for your interest in this project. We could not merge this PR due to the conflicts raised and extremely sorry for the fact that we could not wait much to get the relevant changes from you as we were on a hurry during the time to do a release on the same. But do Keep in touch with us. Hope to see many more new contributions from you in future.

**Pull request closed without merging**